src/ORM/Locator/LocatorInterface.php

```diff
@@ -24,17 +24,6 @@
  */
 interface LocatorInterface
 {
-
-    /**
-     * Stores a list of options to be used when instantiating an object
-     * with a matching alias.
-     *
-     * @param string|null $alias Name of the alias
-     * @param array|null $options list of options for the alias
-     * @return array The config data.
-     */
-    public function config($alias = null, $options = null);
```

Comment:
Is this class on your list of classes that need `@method` turned into real methods?

Reply:
Oui monsieur https://github.com/cakephp/cakephp/issues/12094